scripts/schema-manager/setup.ts

```diff
@@ -0,0 +1,49 @@
+import { Tigris, TigrisClientConfig } from '@tigrisdata/core';
+import { loadEnvConfig } from '@next/env';
+import * as Log from './logger';
+import { loadTigrisManifest } from './manifest-loader';
+
+// Run the config loader only when not executing within next runtime
+if (process.env.NODE_ENV === undefined) {
+  if (process.env.APP_ENV === 'development') {
+    loadEnvConfig(process.cwd(), true);
+  } else if (process.env.APP_ENV === 'production') {
+    loadEnvConfig(process.cwd());
+  }
+}
+
+async function main() {
+  const tigrisFileManifest = loadTigrisManifest();
+  if (!process.env.TIGRIS_URI) {
+    Log.error('Cannot find TIGRIS_URI environment variable ');
+    process.exit(1);
+  }
+  // setup client
+  const tigrisUri = process.env.TIGRIS_URI;
+  const clientConfig: TigrisClientConfig = { serverUrl: tigrisUri };
+
+  if (process.env.TIGRIS_CLIENT_ID) {
+    clientConfig.clientId = process.env.TIGRIS_CLIENT_ID;
+  }
+  if (process.env.TIGRIS_CLIENT_SECRET) {
```

Comment:
Since the client is able to read the environment variables automatically, do we need this?

Reply:
Needs a release:
https://github.com/tigrisdata/tigris-client-ts/pull/143

Reply:
Ack. Build is failing for that release

Reply:
> Ack. Build is failing for that release

Will have to force push this commit, one release plugin was removed